pycroft/lib/traffic.py

```diff
@@ -151,3 +155,30 @@ def reset_credit(user, processor, target_amount=1*1024**3):
     log_user_event(deferred_gettext("Traffic has ben compensated to 1GiB").to_json(),
                    author=processor,
                    user=user)
+
+
+@with_transaction
+def handle_exceeded_traffic_limits():
```

Comment:
As a rule of thumb, `handle` is not a good name for describing an action. It could mean anything, and furthermore, you don't even describe what object is subject to handling (the user, which gets or a property).

I propose `sync_traffic_blockages` and adding a docstring (better two for blocking/unblocking) describing the technicalities (a membership in $foo is added if and only if…).

Reply:
I wanted to avoid the word "block" because of what shreyder [wrote at issue 165](https://github.com/agdsn/pycroft/issues/165#issuecomment-372234114).